src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -92,6 +94,32 @@ package
     }
     
     /**
+     * Send messages to the browser console log.In future this can be hooked to any other Debugging tools.
+     * @private
+     * @method AkamaiHDPlayer#SendToDebugger
+     * @param {string} value The value to be passed to the browser console.
+     * @param {string} referrer The fuction or process which passed the value.
+     * @param {string} channelBranch It can be info, debug, warn, error or log.
+     * @returns {boolean} True or false indicating success
+     */
+    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "log"):Boolean
+    {
+      var channel:String; 
+      if (channelBranch == "log") 
+      {
+        channel = "OO." + channelBranch;
+      }
+      else 
+      {
+        channel = "console.log";
```

Comment:
So we shouldn't actually be writing to console.log ever.  You can lot go console.error and console.warn, but if the message is type log it should be OO.log.  Looks like you are mapping all non-log messages back to console.log?  Maybe try "console." + channelBranch